### PR TITLE
fix(banner): add missing custom modifier

### LINKED ISSUE #6140: Penta - Banner is missing custom style modifier
Followup to https://github.com/patternfly/patternfly/pull/6081

It seems that I missed adding a modifier for the custom variant of the banner.

### PR DESCRIPTION
**Pull request by wise-king-sullyman**
Closes #6140 

Convenience link: https://patternfly-pr-6141.surge.sh/components/banner/#status

### CODE REVIEW

**Review by srambach — APPROVED** [review]
Good catch!

File: src/patternfly/components/Banner/banner.scss [hunk 1]
```diff
@@ -90,6 +90,11 @@
     --#{$banner}--Color: var(--#{$banner}--m-info--Color);
   }
 
+  &.pf-m-custom {
+    --#{$banner}--BackgroundColor: var(--#{$banner}--m-custom--BackgroundColor);
+    --#{$banner}--Color: var(--#{$banner}--m-custom--Color);
```
Comment (srambach) [inline src/patternfly/components/Banner/banner.scss:95]: I believe all these colors should be using the "on-" tokens - e.g. `--global-text-color-status-on-custom-default`

Comment (wise-king-sullyman) [reply src/patternfly/components/Banner/banner.scss:95]: I'm probably misunderstanding something, but I assign those tokens to the modifier vars [higher in the file](https://github.com/patternfly/patternfly/blob/12db7dae8e8c7a01be8be6d78484d46406a4c9e3/src/patternfly/components/Banner/banner.scss#L41). Do I need to redo something about how I did that?

Comment (srambach) [reply src/patternfly/components/Banner/banner.scss:95]: No I'm just an idiot today. Sorry, trying to do too many things at once.

**Review by srambach — APPROVED** [review]
For reals, looks great.

**Review by mcoker — APPROVED** [review]
🍰

**Pull request merged**